src/main/java/com/uber/jenkins/phabricator/PhabricatorBuildWrapper.java

```diff
@@ -54,8 +54,11 @@ public Environment setUp(AbstractBuild build,
             return this.ignoreBuild(logger, "No environment variables found?!");
         }
 
+        final String arcPath = this.getArcPath();
+
         final Map<String, String> envAdditions = new HashMap<String, String>();
         envAdditions.put(PhabricatorPlugin.WRAP_KEY, "true");
+        envAdditions.put(PhabricatorPlugin.ARCANIST_PATH, arcPath);
```

Comment:
i really wish there was a better way to share data between the two plugins, since this repo is essentially a build wrapper (environment setup) and a notifier (record the result), but i don't want people to have to configure things twice in the global configuration. is there a better pattern for this?